src/pages/AttendancePage.tsx

```
@@ -24,6 +24,7 @@ import {
     SelectTrigger,
     SelectValue,
 } from "@/components/ui/select";
+import { useToast } from "@/components/ui/useToast";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useToast'. Consider to delete it or rename it.

'useToast' is defined here:


(lint/suspicious/noRedeclare)